### DISCUSSION TITLE
Wallet Registry pending - How much time will take for approval

### DISCUSSION THREAD

**Discussion by kumarnagender09** [WalletConnect/walletconnect-monorepo#943] [Developer Support]
Hi 
I have submitted an application to register my wallet but it has been pending review for 1 hour.
Could you approve it

**Comment by kumarnagender09** [comment]
My app name is Clarity

**Reply by chadyj** [reply]
Responded.

**Reply by kumarnagender09** [reply]
Thanks for reply 
Before live : I want to test this on testflight. Is it possible?

**Reply by chadyj** [reply]
Yes, you don't need the registry to test most functions.

**Reply by kumarnagender09** [reply]
I am developing the iOS native application and  I want to show the list of wallets in my app > user will choose and do the signin in app.

But currently the app is not showing the wallet options screen. Please let me know how to achieve this. 

<img width="436" alt="Screenshot 2022-04-13 at 2 57 28 PM" src="https://user-images.githubusercontent.com/1977940/163145548-1a3bb073-7d63-4515-a26c-098cf09ce95d.png">

**Reply by chadyj** [reply]
You have the option of creating this UI using the registry API.
See https://docs.walletconnect.com/2.0/api/registry-api

**Reply by kumarnagender09** [reply]
Wallet connect SDK will not provide the same UI?

**Reply by kumarnagender09** [reply]
@chadyj  :  Every-time I am getting an error with rainbow wallet. 

"Error Initialising with walletConnect"

deep link : "wc://wc?uri=wc:D9CC6090-062A-4B51-B46C-B6313526D643@1?bridge=https%3A%2F%2Fsafe-walletconnect.gnosis.io%2F&key=aff83f171a54882bbb08a407fc570e2a5c719a2d514e37e9bacd44274cda5f71"

However it is working  fine with MetaMask.

could you please help me why this is happening  and how to fix this?

![IMG_569E0F30F405-1](https://user-images.githubusercontent.com/1977940/163166919-301ee686-ee2e-49ea-9ba6-44d1af417f9a.jpeg)
Mask
![IMG_8983](https://user-images.githubusercontent.com/1977940/163166864-d85757d7-6a26-4233-8e13-3217bb7b096b.PNG)

**Reply by kumarnagender09** [reply]
wc://wc?uri=wc:D9CC6090-062A-4B51-B46C-B6313526D643@1?bridge=https%3A%2F%2Fsafe-walletconnect.gnosis.io%2F&key=aff83f171a54882bbb08a407fc570e2a5c719a2d514e37e9bacd44274cda5f71"

**Reply by kumarnagender09** [reply]
Still stuck here
